service/files/defaults.yaml

```diff
@@ -7,13 +7,8 @@ configs:
     wsgi:
       workers: 4
 
-    sources:
-      - region: "hooli-west-1"
-        elastic_src: "http://127.0.0.1:9200/log-*/log"
-
-    backend:
-      elastic: "http://127.0.0.1:9200"
-      elastic_index: "ms_health_idx_1"
+    sources: []
+    elastic_index: "ms_health_idx_1"
```

Comment:
Okay, I removed this index from conf files. Yes, the backend parameter is still used and it sets in `service/files/health.json.j2`, please take a look on it.